app/api/projects/explore/route.ts

```
@@ -12,7 +12,8 @@ export async function GET(request: NextRequest) {
     const skip = parseInt(searchParams.get('skip') || '0')
 
     const db = await getDatabase()
-    const query: any = { isPublic: true }
+    // exclude drafts from public explore
+    const query: any = { isPublic: true, isDraft: { $ne: true } }
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)